lib/statsample-glm/glm/formula.rb

```diff
@@ -29,9 +29,11 @@ def parse_formula(form = :token)
 
       private
 
-      def add_contant_term_if_required
-        # TODO: Add support for constants
-        @tokens.unshift Token.new('1')
+      def manage_contant_term
+        @tokens.unshift Token.new('1') unless
+          @tokens.include?(Token.new('1')) ||
+          @tokens.include?(Token.new('0'))
+        @tokens.delete Token.new('0')
```

Comment:
I decided to split `FormulaWrapper` form `Token` classes into different files because `FormulaWrapper` is going to get big after including support for symbols '*' and '/'. So, is there a way to make it work in this scenario?